.vuepress/config.js

```diff
@@ -1,4 +1,4 @@
-const path = require('path');
+const path = require("path");
```

Comment:
I suggest to not add any linting just now because there are a lot of competing PRs to modify this file. I don't see anything related to IPFS in it, correct?

Reply:
I've added the link to the tutorials here: https://github.com/cosmos/sdk-tutorials/pull/1207/files#diff-d922ba5eb323770279ac8c0f81af0d64178cee0268702f003879abc3e5260ffdR149-R154.
I feel like we should all use the same linter and linting this file makes it easier to read.